helm/resource_helm_release.go

```diff
@@ -497,6 +499,12 @@ func (r *HelmRelease) Schema(ctx context.Context, req resource.SchemaRequest, re
 				Computed:    true,
 				Description: "Status of the release",
 			},
+			"take_ownership": schema.BoolAttribute{
+				Optional:    true,
+				Computed:    true,
+				Default:     booldefault.StaticBool(false),
```

Comment:
Would it make sense to use a value from the `defaultAttributes` map here? Maybe in a few other places in this PR too?